### DISCUSSION TITLE
offscreen - ScreenshotOrNull() transparent background

### DISCUSSION THREAD

**Discussion by luqinghui** [cefsharp/CefSharp#3840] [General]
i use the ScreenshotOrNull() to get a snapshot of [mapbox](https://github.com/mapbox/mapbox-gl-js) map,
when i set the white background, 
![image](https://user-images.githubusercontent.com/5338140/136642734-359cc056-1540-4d2e-93f7-45ac80fddca1.png)
but when i set the transparent background, text and line seems blurry
![image](https://user-images.githubusercontent.com/5338140/136642852-327cb764-baa2-4cfb-bcb6-95c5599d1084.png)
when i set a white halo for text, it seems normal
![image](https://user-images.githubusercontent.com/5338140/136642911-9390428c-5900-4e7b-92c5-16536139d6f6.png)
is this a bug? anyone konw how to fix it? thanks😊

**Comment by amaitland** [comment]
Make sure your application has an app.manifest and are using version 94.4.50

https://github.com/cefsharp/CefSharp.MinimalExample/blob/master/CefSharp.MinimalExample.OffScreen/app.manifest

>  is this a bug? 

Impossible to say as you haven't provided any details. 

It's important to remember that CefSharp is just one of many chromium embedded framework wrappers. The actual rendering is done in CEF/Chromium.
